src/radical/saga/adaptors/redis/redis_advert.py

```diff
@@ -232,7 +232,7 @@ def attribute_getter (self, key) :
 
         except Exception as e :
             self._logger.error ("get_key failed: %s" % e)
-            raise e
+            raise
```

Comment:
Wouldn't it make sense to remove the log and keep the exception message in the raise?

Reply:
`raise e` does the same as `raise` - a raise w/o explicit exception raised the current exception stack.  We use that syntax elsewhere and it is the recommended syntax (https://www.python.org/dev/peps/pep-3109/), thus the change.